### ISSUE TITLE
[Bug]: filter boolean with tablenames throws an error

### ISSUE THREAD

**Issue by marineusde** [Power-Components/livewire-powergrid#1037]
### Have you searched through other issues to see if your problem is already reported or has been fixed?

Yes, I did not find it.

### Did you read the documentation?

Yes, I did not find it.

### Have you tried to publish the views?

No, this error is not related to views.

### Is there an error in the console?

 foreach() argument must be of type array|object, string given
 
found the foreach in:

Illuminate\Support\Arr Line 114

### PHP Version

8.1.20

### PowerGrid

4.8

### Laravel

10.13.5

### Livewire

2.12.3

### Alpine JS

3.12

### Theme

Bootstrap

### Describe the bug.

If I use a table name in a filter, there is a bug after filtering the boolean to "yes" or "no".

In the trait PowerComponents\LivewirePowerGrid\Traits\HasFilter in the method filterBoolean() in line 315 the array $this->filters['boolean'] is already set with:

```
[
  "tablename" => array:1 [▶
    "is_active" => "true"
  ]
]
```

After setting the value its (Line 317):
```
[
  "tablename" => array:1 [▶
    "is_active" => "true"
  ]
  "tablename.is_active" => "true"
]
```

The problem is, that "tablename.is_active" is a string, not an array so the Collection-Filter crashes.

Its NOT there if you don't use table names, but that's not a good solution.

### To Reproduce...

```
public function columns(): array
{
    return $this->setColumnArray([
        Column::make('Status', 'is_active')
            ->toggleable(),
        Column::make('Name','name')
            ->sortable(),
    ]);
}

public function filters(): array
{
    return [
        Filter::boolean('is_active', 'tablename.is_active')
    ];
}
```

### Extra information

```markdown
I will fix it in a branch
```

**Comment by marineusde** [comment]
Fixed in #1038

**Comment by luanfreitasdev** [comment]
Thank you!.
